qovery/resource_cluster_model.go

```diff
@@ -182,13 +182,23 @@ func fromQoveryClusterFeatures(ff []qovery.ClusterFeature) types.Object {
 		}
 	}
 
+	// The object should be fill even if no feature is present, but will be mark as Null
+	// (e.g SCW clusters don't have any feature)
+	isNull := false
 	if len(attrs) == 0 && len(attrTypes) == 0 {
-		return types.Object{Unknown: true}
+		isNull = true
+		defaultFeatureKeyVpcSubnet := ""
+		defaultFeatureKeyStaticIP := false
```

Comment:
Is there a way to avoid declaring those variables ? 
Wanted to do a kind of `attrs[featureKeyVpcSubnet] = fromStringPointer("value".to_ref())` like in rust x)